force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -176,7 +225,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The average of the three numbers.
      */
     public static Double averageOfThreeNumbers(Integer a, Integer b, Integer c) {
-        return null; // Replace null with the variable you used to store the result
+        Double sum = 0;
+        if(a != null && b != null && c != null){
```

Comment:
What if a is null and b and c have values, Would your code not add b & c? I'm not saying you should or should not, just something to think about.